docs/content/FAQ/_index.md

```diff
@@ -35,6 +36,23 @@ If you want to see what has changed between versions, refer to the [releases pag
 
 The Monkey shuts off either when it can't find new victims, or when it has exceeded the quota of victims as defined in the configuration.
 
+## How to reset the password?
+
+On your first access of Monkey Island server, you'll be prompted to create an account. If you forgot the credentials you
+ entered or just want to change them, you need to manually alter the `server_config.json` file. On Linux, this file is 
+ located on `/var/monkey/monkey_island/cc/server_config.json`. On windows, it's based on your install directory (typically 
+ `C:\Program Files\Guardicore\Monkey Island\monkey_island\cc\server_config.json`). Reset the contents of this file back to
+
+```json
+{
+  "server_config": "password",
+  "deployment": "windows"
+}
+```
```

Comment:
Would rather not give the example with a deployment option that may be incorrect.
Better to tell them what keys to delete.

Reply:
?

Reply:
Addressed this by specifying in bold, to leave deployment option unchanged. I don't think reading a list of keys (which we'll have to update) is easier than just copy pasting the code and then copy pasting the previous deployment.

Reply:
Agreeing